cmd/cli.go

```diff
@@ -158,6 +158,16 @@ func (cli *CLI) Run(args []string) int {
 	}
 	runners = append(runners, runner)
 
+	var ruleNames []string
+	for name := range cfg.Rules {
+		ruleNames = append(ruleNames, name)
+	}
+	err = rules.CheckRuleNames(ruleNames)
+	if err != nil {
+		formatter.Print(tflint.Issues{}, tflint.NewContextError("Failed to check rule config", err), cli.loader.Sources())
+		return ExitCodeError
```

Comment:
It would be better to add a test case for the error like the following:
https://github.com/wata727/tflint/blob/96dd0d36f38515b8def4d4ccb37ff31da28bcb7c/cmd/cli_test.go#L97-L102

Reply:
Done.